tests/examples_e2e/setup_module/main.tf

```diff
@@ -200,6 +201,11 @@ resource "local_file" "terraform_tfvars" {
       email     = google_service_account.service_account.email
       iam_email = "serviceAccount:${google_service_account.service_account.email}"
     }
+    service_identities = {
+      secret = {
+        email = resource.google_project_service_identity.jit_si["secretmanager.googleapis.com"].email
+      }
+    }
```

Comment:
I'm reluctant about adding those here, as we can do that only for JIT-ed service accounts, for all others we need to generate the name by hand. 

So I'd prefer to create a identity address by hand using `var.project_number`